blog/.vuepress/config.js

```diff
@@ -38,7 +38,7 @@ module.exports = {
       }
     ]
   },
-  title: "",
+  title: "VueMtl",
```

Comment:
I'd propose MTL (like the slack channel)
```suggestion
  title: "VueMTL",
```

@tlebel what's the official name case?

Reply:
Good question! Actually the name is "Vue Montreal". 
Tshirts and logo read VUE MTL for shorter version... I know a bit incoherent. 

The slack is also "https://vuemontreal.slack.com", which I just updated the actual name to avoid confusion! 

"Vue Montreal" is less cryptic and better on the SEO side in the mean time!